src/Controllers/ElementModelAdmin.php

```diff
@@ -83,14 +136,12 @@ public function getEditForm($id = null, $fields = null)
         $dc = $gf->getConfig()->getComponentByType(GridFieldDataColumns::class);
         if ($dc) {
             $display_fields = [
-                'ID' => _t('ElementalModelAdmin.NUM', '#'),
                 'Title' => _t('ElementalModelAdmin.TITLE','Title'),
                 'Parent.OwnerTitleAndDescription' => _t('ElementalModelAdmin.CONTEXT','Context'),
                 'Type' => _t('ElementalModelAdmin.TYPE','Type'),
-                'LastEdited.Nice' => _t('ElementalModelAdmin.EDITED','Edited'),
```

Comment:
We need to know when the last edited and created date was, exactly. Ago will show something like "38 hours ago".